package.json

```diff
@@ -44,18 +54,23 @@
     "xpath": "0.0.27"
   },
   "devDependencies": {
+    "@types/dotenv": "^6.1.1",
```

Comment:
some dependencies have ^ before version, some have strict version - I think we should accept one solution in this issue